modules/mono/editor/godotsharp_builds.cpp

```diff
@@ -366,6 +367,25 @@ bool GodotSharpBuilds::build_project_blocking(const String &p_config) {
 	pr.step("Building project solution", 0);
 
 	MonoBuildInfo build_info(GodotSharpDirs::get_project_sln_path(), p_config);
+
+	// Add OS defines
+	String constants = "Constants=\"";
+
+	constants += "GODOT_" + OS::get_singleton()->get_name().to_upper() + ";"; // OS
+
+	if (sizeof(void *) == 4) {
+		constants += "GODOT_32BIT;";
+	} else {
+		constants += "GODOT_64BIT;";
+	}
```

Comment:
Both OS and bit constants are fine when building for the editor and editor player, but not when exporting the game. It would need passing this info to `build_project_blocking` from `GodotSharpExport::_export_begin`.

Reply:
Done